proto/tikvpb.proto

```diff
@@ -146,12 +138,8 @@ message BatchCommandsRequest {
             kvrpcpb.CheckTxnStatusRequest CheckTxnStatus = 25;
             kvrpcpb.TxnHeartBeatRequest TxnHeartBeat = 26;
 
-            kvrpcpb.VerGetRequest VerGet = 27;
-            kvrpcpb.VerBatchGetRequest VerBatchGet = 28;
-            kvrpcpb.VerMutRequest VerMut = 29;
-            kvrpcpb.VerBatchMutRequest VerBatchMut = 30;
-            kvrpcpb.VerScanRequest VerScan = 31;
-            kvrpcpb.VerDeleteRangeRequest VerDeleteRange = 32;
+            // Deprecated for VerKV
+            // reserved 27 to 32;
```

Comment:
Why comment it?

Reply:
`reserved` is just valid for `message` and unsupported for `oneof`. Protoc cannot understand the comment, but at least this comment can remind humans.

Reply:
You can put it right after declaration of `oneof` inside the same message.

Reply:
Interesting... Refresh my knowledge. I will handle it soon.